lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_retrospectives.py

```diff
@@ -471,7 +472,9 @@ def create_encounter_status(redcap_record: dict) -> str:
 
     standardized_status = standardize_whitespace(status.lower())
 
-    if standardized_status not in mapper:
+    if standardized_status in mapper.values():
+        return standardized_status
+    elif standardized_status not in mapper:
         raise Exception(f"Unknown encounter status «{standardized_status}».")
 
     return mapper[standardized_status]
```

Comment:
I'm not sure I see the motivation for this change - is there a reason to not let the existing `return mapper[standardized_status]` handle it? With the change, I'm not sure it will ever reach line 480.

Reply:
It checks for a match in the map values before checking the map keys (it will get to line 480 if its in keys). I had Ben add a couple of additional fields to the ndjson file to avoid hardcoding them in the ETL, one of them being `"encounter_status":"finished"`, which really doesn't need to be mapped but I included it just for the validation piece. There are other mapping functions that use a similar approach, checking map values before keys:
https://github.com/seattleflu/id3c-customizations/blob/2248106401316b688d846421f2de42d6ea36c5dc/lib/seattleflu/id3c/cli/command/etl/clinical.py#L543

Reply:
Thanks Dave! I see what you're doing here with the values now. Thanks!